apps/backoffice-v2/src/domains/entities/hooks/useFilterEntity/useFilterEntity.tsx

```diff
@@ -1,7 +1,7 @@
-import { useSearchParamsByEntity } from '../../../../common/hooks/useSearchParamsByEntity/useSearchParamsByEntity';
+import { useEntityType } from '../../../../common/hooks/useEntityType/useEntityType';
 
-export const useFilterEntity = (): string | undefined => {
-  const [{ entity }] = useSearchParamsByEntity();
+export const useFilterEntity = () => {
+  const entity = useEntityType();
```

Comment:
Any reason not to use `useEntityType` directly?

Reply:
@Omri-Levy Practically there is no point of using `useFilterEntity` in current context when it servers only entity type.But its been as good isolation level to context of `entities` features and in case if its behaviour will be extended we don't have to update all hooks/components which used it.

Updated code,removed `useFilterEntity` hook